orange/src/main/java/com/hartwig/hmftools/orange/report/PageEventHandler.java

```diff
@@ -67,7 +68,7 @@ void resetChapterPageCounter() {
     }
 
     void writeTotalPageCount(@NotNull PdfDocument document) {
```

Comment:
Not related to this actual change, but shouldn't this function be called `writeFooters`? Your renames in this class are good renames but suspect you missed one :)